doc/development/administration.rst

```diff
@@ -405,22 +406,27 @@ each release.
     and pass the CI.
 #.  If releasing a backported version branch, cherry-pick this documentation
     commit into the backport branch.
-#.  Use the GitHub interface to create a new release with the documentation
-    update that was just added.
+#.  Use the release-notes `notebook`_ to push a draft release to GitHub.
+#.  Using the GitHub interface, verify the release notes and publish it.
+
+.. _notebook: https://github.com/celeritas-project/release-scripts
```

Comment:
Site note: the notebook appears to fail at least for me:
For https://github.com/celeritas-project/release-scripts/blob/main/release-notes.ipynb
```
TypeError                                 Traceback (most recent call last)
Cell In[45], line 1
----> 1 artifact_url, artifact_tgz = release_notes.get_or_upload_tarball(cached, release)
      2 print(f"Artifact URL: {artifact_url}")
...
TypeError: 'NoneType' object is not subscriptable
````